requirements-dev.txt

```diff
@@ -1,131 +1,135 @@
 alabaster==0.7.13
-astroid==2.12.13
-attrs==22.1.0
+astroid==2.15.8
 autoflake==1.7.8
-Babel==2.11.0
-beautifulsoup4==4.11.1
-black==22.12.0
-bleach==5.0.1
-cachetools==5.2.0
-certifi==2022.12.7
-cffi==1.15.1
-chardet==5.1.0
-charset-normalizer==2.1.1
-click==8.1.6
+Babel==2.12.1
+beautifulsoup4==4.12.2
+black==23.9.1
+bracex==2.4
+cachetools==5.3.1
+certifi==2023.7.22
+cffi==1.16.0
+chardet==5.2.0
+charset-normalizer==3.2.0
+click==8.1.7
 codespell==2.2.5
 colorama==0.4.6
-commonmark==0.9.1
-coverage==6.5.0
+coverage==7.3.1
 craft-archives==1.1.3
 craft-cli==2.1.0
 craft-parts==1.25.1
-craft-providers==1.16.0
-cryptography==38.0.4
+craft-providers==1.18.0
+cryptography==41.0.4
 Deprecated==1.2.14
 dill==0.3.7
 distlib==0.3.7
 distro==1.8.0
-docutils==0.17.1
-exceptiongroup==1.0.4
-filelock==3.8.2
+docutils==0.18.1
+exceptiongroup==1.1.3
+filelock==3.12.4
 flake8==4.0.1
-furo==2022.12.7
-gnupg==2.3.1
```

Comment:
how come this got dropped?

Reply:
It was an indirect dependency via craft-archives, but we removed that dependency from craft-archives back in May.